Gemfile

```diff
@@ -9,6 +9,12 @@ gem 'rake'
 gem 'rack-conneg', '~> 0.1.5'
 gem 'rack-rest_api_versioning', '~> 0.0.2'
 
+gem 'google-api-client'
+
+#gem "foreman", :path => "/Users/pje/my_foreman_fork"
+#gem "GGB", :path => "/Users/dlhaines/dev/BITBUCKET/ggb"
+gem "GGB", :path => "/Users/dlhaines/dev/GITHUB/dlh-umich.edu/FORKS/google_groups_gem"
```

Comment:
I was going to suggest that if there's a typical path that gems use for this path, like "/usr/lib/share", use that path here.  On your computer, symlink "/usr/lib/share/google_groups_gem" to the path you have above.

Also, remove commented-out lines.